repl/src/test/scala/org/apache/spark/repl/ExecutorClassLoaderSuite.scala

```diff
@@ -99,6 +107,32 @@ class ExecutorClassLoaderSuite
     }
   }
 
+  test("resource from parent") {
+    val parentLoader = new URLClassLoader(urls2, null)
+    val classLoader = new ExecutorClassLoader(new SparkConf(), url1, parentLoader, true)
+    val resourceName: String = "fake-resource.txt"
```

Comment:
Better for this to be a field in the class since you're also using it above in `parentResourceNames`.